src/code/templates/fv-genome-challenge.js

```diff
@@ -49,13 +54,36 @@ class YourDrakeView extends React.Component {
   }
 }
 
+/*
+ * TargetDrakeView
```

Comment:
Certainly not necessary, but when there are multiple components in a file, I think it helps to have visual breaks for scrolling/navigation. They also provide a place for comments about the role of the subcomponents, although I'm not using them for that purpose in this case.